backend/src/db/migrations/20251031044512_add-certificate-sync-external-identifier.ts

```diff
@@ -0,0 +1,21 @@
+import { Knex } from "knex";
+
+import { TableName } from "../schemas";
+
+export async function up(knex: Knex): Promise<void> {
+  if (!(await knex.schema.hasColumn(TableName.CertificateSync, "externalIdentifier"))) {
+    await knex.schema.alterTable(TableName.CertificateSync, (t) => {
+      t.text("externalIdentifier").nullable();
```

Comment:
can we just use `t.string()` here? I think we use `text` if the values can get pretty long like descriptions etc.